packages/controllers/src/plugins/PluginController.ts

```diff
@@ -134,40 +149,56 @@ export class PluginController extends EventEmitter {
 
   private _hasPermission: HasPermissionFunction;
 
+  private _terminateWorkerOf: TerminateWorkerOf;
+
+  private _command: Command;
+
+  private _terminateAll: TerminateAll;
+
+  private _createPluginWorker: CreatePluginWorker;
+
+  private _startPlugin: StartPlugin;
+
   private _pluginsBeingAdded: Map<string, Promise<Plugin>>;
 
   constructor({
-    initState,
-    setupWorkerPluginProvider,
     removeAllPermissionsFor,
     closeAllConnections,
     requestPermissions,
     getPermissions,
+    terminateWorkerOf,
+    terminateAll,
     hasPermission,
-    workerUrl,
+    createPluginWorker,
+    startPlugin,
+    command,
+    messenger,
+    state,
   }: PluginControllerArgs) {
-    super();
-    const _initState: PluginControllerState = {
-      plugins: {},
-      pluginStates: {},
-      ...initState,
-    };
-
-    this.store = new ObservableStore({
-      plugins: {},
-      pluginStates: {},
-    });
-
-    this.memStore = new ObservableStore({
-      inlinePluginIsRunning: false,
-      plugins: {},
-      pluginStates: {},
+    super({
+      messenger,
+      metadata: {
+        inlinePluginIsRunning: {
+          persist: false,
+          anonymous: false,
```

Comment:
We could probably make this `true` here? This isn't exactly personally identifiable. Though I'd imagine this is not here long-term, since it's only for testing.